src/azure/monitor/opentelemetry/distro/distro.py

```diff
@@ -0,0 +1,21 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from os import environ
+
+from opentelemetry.environment_variables import (OTEL_LOGS_EXPORTER,
+                                                 OTEL_METRICS_EXPORTER,
+                                                 OTEL_TRACES_EXPORTER)
+from opentelemetry.instrumentation.distro import BaseDistro
+
+
+class AzureMonitorDistro(BaseDistro):
+    def _configure(self, **kwargs):
+        environ.setdefault(OTEL_LOGS_EXPORTER, "azure_monitor_opentelemetry_exporter")
```

Comment:
Let's comment this out and leave a TODO until logging is not in preview anymore.

Reply:
Will do